libraries/chain/account_object.cpp

```diff
@@ -138,7 +138,6 @@ set<address> account_member_index::get_address_members(const account_object& a)c
       result.insert(auth.first);
    for( auto auth : a.active.address_auths )
       result.insert(auth.first);
-   result.insert( a.options.memo_key );
```

Comment:
why?

Reply:
Because memo key is not an address. I guess it's here due to copied code from `get_key_members()`.

Actually, the result `account_to_address_memberships` is only used in `get_key_references` database API, memo key will be processed when iterating `account_to_key_memberships` even if it's not added here: https://github.com/bitshares/bitshares-core/blob/d94cc574b2ca460dd636e573f25f0c7737b4a355/libraries/app/database_api.cpp#L558-L562

Do you think it's better to keep the behavior unchanged in this PR, and submit an new issue for the cleanup?

Reply:
Makes sense, thanks. Keep it here.